### DISCUSSION TITLE
Adobe Animate 2020 will not package Harman AIR for Desktop 33.1.1.190 application using self-signed cert as Windows Installer

### DISCUSSION THREAD

**Discussion by tktktk45** [airsdk/Adobe-Runtime-Support#631] [Q&A]
Error creating files.
Unknown error.
unexpected failure: Platform conversion failed. Process exited with error 5. Output was "[ErrorEventtype= "error" bubbles=false cancelable=false eventPhase=2text= "execute: UPDATE 'Property' SET'Valuer ? WHERE 'Property'= ?[1627]: MSI Error" errorlD=0]
it
java.io.lOException: Platform conversion failed. Process exited with error 5. Output was "[ErrorEventtype= "error" bubbles=false cancelable=false eventPhase=2 text= "execute: UPDATE 'Property' SET 'Valuer ? WHERE 'Property'= ?[1627]: MSI Error" errorlD=0]
it
at
com.adobe.air.nai.NativePackager.conversionFail(NativePackager.java)
at
com.adobe.air.nai.NativePackager.invokePlatformConversion(NativePa
ckager.java)
at
com.adobe.air.nai.NativePackager.createPackage(NativePackager.java) at com.adobe.air.ADT.parseArgsAndGo(ADT.java) at com.adobe.air.ADT.run(ADT.java) at com.adobe.air.ADT.main(ADT.java)

![2020-08-12_10-19-16](https://user-images.githubusercontent.com/8018386/90034829-ba724200-dc86-11ea-9f0f-64117993238f.png)

**Comment by patrickdadzio** [comment]
Hello,

I have the same problem. Did you find a solution to this problem?

Thanks,
Patrick.

**Reply by patrickdadzio** [reply]
I was able to solve my problem :-) I had a pretty big folder with resources that were added to the build. I removed it and add it in the installer later.
